example/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "This folder contains an example configuration file and custom rules for Schemalint.",
+  "private": true,
+  "scripts": {
+    "start-example-db": "docker run -d --name dvd-rental -p 54321:5432 kristiandupont/dvdrental-image",
+    "stop-example-db": "docker stop dvd-rental",
+    "lint:schema": "schemalint"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "schemalint": "^1.1.0"
```

Comment:
Maybe this should just be version `"*"`?

Reply:
I agree with you. Using `"*"` would save us the trouble of keeping up with the latest version. Update in https://github.com/kristiandupont/schemalint/pull/340/commits/954dd4817a2743f85517766cae1432674a2118dc.